### PR TITLE
feat: a/b test lock icon

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This pr adds a security lock icon to the donation description and the donation cta. 

How to test:
A variant: seed the db, login, check the email (foo@bar.com), and pop the donation modal. It should not include lock icons.
B variant: seed the db, login, change the email to (foo@foo.com), and pop the donation modal It should have two lock icons.

B variant screenshot of the donation modal:

Screen Shot 2022-03-22 at 10 40 57 AM

### CODE REVIEW

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -306,6 +306,7 @@
     "runs-in-vm": "The project runs in a virtual machine, complete the user stories described in there and get all the tests to pass to finish step 1."
   },
   "donate": {
+    "donate-cta":"Donate",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/translations.json:309]: Missing space here, but I am not that fussed.

Also, we do technically have the word `Donate` already in the translations.

**Review by ShaunSHamilton — DISMISSED** [review]
LGTM 👍 

---

I did not get the button up, but did see the lock in the text:
image

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (ahmadabdolsaheb) [inline client/i18n/locales/english/translations.json:309]:
```suggestion
    "donate-cta": "Donate",
```

File: client/src/components/Donation/stripe-card-form.tsx [hunk 2]
```diff
@@ -167,7 +170,8 @@ const StripeCardForm = ({
         disabled={!stripe || !elements || isSubmitting}
         type='submit'
       >
-        Donate
+        {isAVariant === false && <SecurityLockIcon />}
+        {t('donate.donate-cta')}
```
Comment (ahmadabdolsaheb) [inline client/src/components/Donation/stripe-card-form.tsx:174]:
```suggestion
        {t('buttons.donate')}
```

File: client/i18n/locales/english/translations.json [hunk 3]
```diff
@@ -306,6 +306,7 @@
     "runs-in-vm": "The project runs in a virtual machine, complete the user stories described in there and get all the tests to pass to finish step 1."
   },
   "donate": {
+    "donate-cta": "Donate",
```
Comment (ahmadabdolsaheb) [inline client/i18n/locales/english/translations.json:309]:
```suggestion
```

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Pull request merged**